pages/common/atom.md

```diff
@@ -0,0 +1,11 @@
+# atom
+
+> A hackable text editor for the 21st century.
```

Comment:
Maybe a little more descriptive? I.e. explain what makes it unique (configurability, based on x, aimed at devs of kind y...)